programming_examples/makefile-common

```diff
@@ -3,6 +3,8 @@ AIETOOLS_DIR ?= $(shell realpath $(dir $(shell which xchesscc))/../)
 AIE_INCLUDE_DIR ?= ${AIETOOLS_DIR}/data/versal_prod/lib
 AIE2_INCLUDE_DIR ?= ${AIETOOLS_DIR}/data/aie_ml/lib
 
+AIEOPT_DIR ?= $(shell realpath $(dir $(shell which aie-opt))/../)
```

Comment:
```suggestion
AIEOPT_DIR ?= $(shell realpath $(dir $(shell which aie-opt))/..)
```

Reply:
Copy the entire contents of the directory in cmake